### PR TITLE
LOG-6019: Setup CI for Vector release-6.1

### PR DESCRIPTION
**Pull request by vparfonov**
Setup CI for Vector release-6.1

/assign @jcantrill
/cc @cahartma @Clee2691

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
Also create a file like https://github.com/openshift/release/blob/2c2508dcf6ddc46a0d43ed477ab7bd092d8344be/core-services/image-mirroring/openshift-logging/mapping_logging_5_9_quay#L8 and mirror ONLY the vector 6.1 image

**Comment by vparfonov** [comment]
> Also create a file like
> 
> https://github.com/openshift/release/blob/2c2508dcf6ddc46a0d43ed477ab7bd092d8344be/core-services/image-mirroring/openshift-logging/mapping_logging_5_9_quay#L8
> 
> and mirror ONLY the vector 6.1 image

We don't have such file for 6.0. Should i added it?

File: ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      region: us-east-2
+    owner: obs-logging
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.15"
+  steps:
+    test:
+    - as: test
+      cli: latest
+      commands: export GINKGO_OPTS="-ginkgo.skip fluentd"; FAKE_HOME_DIR=/tmp/home
```
Comment (vparfonov) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml:114]: @jcantrill can we remove: `GINKGO_OPTS="-ginkgo.skip fluentd";`?

Comment (jcantrill) [reply ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml:114]: yes

File: ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- as: ldd-openssl
+  commands: ldd /usr/bin/vector | grep -E 'libcrypto|libssl'
+  container:
+    from: vector
+- as: cluster-logging-operator-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
```
Comment (jcantrill) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml:97]: @vparfonov change these to 'us-east-1'

**Review by jcantrill — COMMENTED** [review]
One change for the e2e and functional tests.  I'm hoping the other failure is a flake.

**Comment by jcantrill** [comment]
/pj-rehearse pull-ci-ViaQ-vector-release-6.1-cluster-logging-operator-e2e

**Comment by vparfonov** [comment]
/pj-rehearse pull-ci-ViaQ-vector-release-6.1-cluster-logging-operator-e2e

**Comment by jcantrill** [comment]
/pj-rehearse pull-ci-ViaQ-vector-release-6.1-clo-functional
/pj-rehearse pull-ci-ViaQ-vector-release-6.1-cluster-logging-operator-e2e

**Comment by jcantrill** [comment]
/pj-rehearse pull-ci-ViaQ-vector-release-6.1-clo-functional
/pj-rehearse pull-ci-ViaQ-vector-release-6.1-cluster-logging-operator-e2e

**Comment by jcantrill** [comment]
/pj-rehearse pull-ci-ViaQ-vector-release-6.1-cluster-logging-operator-e2e

File: ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    owner: obs-logging
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.16"
+  steps:
+    test:
+    - as: test
+      cli: latest
+      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=${ARTIFACT_DIR}/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts
+        make test-e2e-olm
```
Comment (jcantrill) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml:107]: change to `test-e2e`

File: ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    owner: obs-logging
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.16"
+  steps:
+    test:
+    - as: test
+      cli: latest
+      commands: FAKE_HOME_DIR=/tmp/home LOG_DIR=${ARTIFACT_DIR}/logging-logs ARTIFACT_DIR=${ARTIFACT_DIR}/logging-artifacts
+        make test-functional-vector
```
Comment (jcantrill) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-release-6.1.yaml:151]: change to `test-functional`

**Pull request merged**